New documentation for chapter 1

kyragaut opened the pull request.

Commit:
first commit of chapter 1: intro to web archiving, autopilot, and installation

frankiebee commented on chapter1.md:
```diff
@@ -0,0 +1,36 @@
+## Status page: pre-made behaviors
+
+
+# Chapter 2: Using Autopilot
```
make chapter 2 a separate file

okdistribute commented on chapter1.md:
```diff
@@ -0,0 +1,36 @@
+Both [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) must be installed to run Autopilot. 
+
+To install Autopilot, use Terminal to clone the Webrecorder repository: 
+1. `git clone `
```
We might want to highlight also building from source; not sure which is best. Is the docker impl. kept up to date? @ikreymer 

This isn't blocking for now though

ikreymer replied:
Yes, Docker implementation should be, it is how its deployed in production for Conifer, will double check that its working. Yeah, probably both should be mentioned.

okdistribute commented on chapter1.md:
```diff
@@ -0,0 +1,36 @@
+
+The Autopilot instance can be accessed in the browser at `http://localhost:8089`.
+
+^^ EXPLAIN BETTER, also do I need to explain remote browsers/what Docker is?
```
Maybe a sentence or two, and/or a link for docker but not much required imo

Commit:
Added file for chapter 2 documentation

Commit:
install section of chapter 1 modified to include a way to install autopilot without Docker

Commit:
fixed in-line code to be code blocks and added some usage info

ikreymer commented:
Just wanted to say this is looking like a great start to more friendly docs :)

Commit:
Added documentation for the usage of autopilot

okdistribute commented on chapter1.md:
````diff
@@ -53,6 +53,66 @@ Commands:
+
+The following options are available to use with the behaviors command:
+
+```
````
as this is intended as a quick getting started guide I think that we don't need to put this output of `help` in here

okdistribute commented on chapter1.md:
````diff
@@ -53,6 +53,66 @@ Commands:
+The following options are available to use with the API command:
+
+```
+$ ./bin/cli api --help
````
same here

okdistribute commented on chapter1.md:
```diff
@@ -53,6 +53,66 @@ Commands:
+* `WR_BEHAVIOR_METADATA_PATH`: path to the behavior metadata file
+* `BUILD_BEHAVIORS`: should the api server build the behaviors before starting
+
 //I can't yet explain how to use autopilot in the browser since the extension isn't finished yet, right? 
```
extension can be discussed in a separate section or added later in the process

Commit:
Added headlines to chapter 2, some usage information for Docker (may not need this), and set up the pre-made behaviors sectin

Commit:
last commit didn't work right, trying again

Commit:
Started the guide on how to write a behavior for people with little js knowledge, still testing the premade behaviors for chapter 1

Commit:
Progress on the Format section for chapter 2 under Creating Your First Behavior

Commit:
Added more for format section, putting in code examples

Commit:
fixing some stuff in format, adding some stuff in a new section explaining metadata

Commit:
wrote some stuff on the expected format of the export metadata object

Commit:
added info on the match property in the metadata export

Commit:
finished the tutorial on how to create a behavior in chapter 2

Commit:
added files for chapters 3, 4, and reference:API

Commit:
added more steps into the "how to make a behavior" section of chapter2 so that
it's easier to follow for a beginner, and some notes in chapter3 for how it
should be written

Commit:
changed the title of the section after "setting up your file" to "getting started using the CLI"

Commit:
added stuff about using the `newBehavior` CLI in the getting started section

Commit:
added directions for Testing your first behavior using the CLI

Commit:
fixed some small formatting things

Commit:
moved chapters to manual

Commit:
wrote directions for how to set up config file in "testing your first behavior"
o Please enter the commit message for your changes. Lines starting

Commit:
wrote the "fixing a broken behavior" section

Commit:
I don't remember what my changes were

Commit:
started the FAQ document

Commit:
moved some advanced stuff from chapter 2 into chapter 3

Commit:
added a code of conduct to the manual

Commit:
removed some stuff that was covered in chapter 2

okdistribute commented on chapter2.md:
```diff
@@ -0,0 +1,299 @@
+# Chapter 2: Creating your first behavior
```
these files need to be in manual or some other sub directory

okdistribute replied:
seems like they're duplicated

Commit:
updated chapter 1

sedmonds commented:
When looking at the section for "Installing Autopilot" and view the ordered list, I'm wondering why we wouldn't remove #3 and #4. Or indicate in some way, steps 3/4 are not needed for basic setup.

ie. Placing these two docker-related steps in an advanced area. 

Let me know if a PR or better clarification would be helpful. Thanks!

Commit:
moved locations of the chapters

Commit:
moved locations of chapters

Commit:
Moved the chapters to the correct locations

Commit:
added bug report format file

okdistribute commented on manual/bug-report.md:
```diff
@@ -0,0 +1,15 @@
+#Bug Report Template
```
this could be in `.github/ISSUE_TEMPLATE/bug-report.md` with the following at the top of the file

```
---
name: Bug Report
about: Report an issue or problem
title: ""
labels: bug
assignees: ''

---
```
